api/nakadi-event-bus-api.yaml

```diff
@@ -1027,6 +1077,11 @@ definitions:
 
   EventTypeSchema:
     properties:
+      version:
+        type: string
+        description: |
+          Automatically generated value based on semantic versioning. Should not be modified by clients.
```

Comment:
OK, I see. So when schema is updated Nakadi checks the changes and generates new version depending on the changes.